RADWIMPS_react.html

```diff
@@ -0,0 +1,59 @@
+<html>
+    <head>
+        <title>RADWIMPS</title>
+        <!-- React Library -->
+        <script 
+            src="https://unpkg.com/react@17.0.2/umd/react.production.min.js"
+            integrity="sha384-7Er69WnAl0+tY5MWEvnQzWHeDFjgHSnlQfDDeWUvv8qlRXtzaF/pNo18Q2aoZNiO"
+            crossorigin
+        ></script>
+        <script
+            src="https://unpkg.com/react-dom@17.0.2/umd/react-dom.production.min.js"
+            integrity="sha384-vj2XpC1SOa8PHrb0YlBqKN7CQzJYO72jz4CkDQ+ePL1pwOV4+dn05rPrbLGUuvCv"
+            crossorigin
+        ></script>
+        <script
+            src="https://unpkg.com/@babel/standalone@7.13.13/babel.min.js"
+            integrity="sha384-5a7c5/GkaCITOh374LENmvBVmw8ts0nszrwqZcCTAhsLvTruncpgJvabCBiaxolO"
+            crossorigin
+        ></script>
+        <style>
+            div {display: inline-block; _display: inline;}
+        </style>
+    </head>
+    <body>
+        <div id="app"></div>
+        <script type="text/babel">
+            class Then extends React.Component{
+                render(){
+                    return(
+                        <div>
+                            前
+                            {this.props.children}
+                        </div>

```

Comment:
レギュレーション的に、DOMの出力に`<div>`が挟まれてしまうのはアウトですのでFragmentを使いましょう。
```suggestion
                        <>
                            前
                            {this.props.children}
                        </>

```